resources/definitions/hellbot_magna_2_230_dual.def.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": 2,
+    "name": "Hellbot Magna 2 230 dual",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Hellbot Development Team",
+        "manufacturer": "Hellbot",
+        "file_formats": "text/x-gcode",
+        "platform": "Hellbot_Magna_2_230.obj",
+        "platform_texture": "Magna2_230.png",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "hellbot_magna_2_230_dual_extruder_0",
+            "1": "hellbot_magna_2_230_dual_extruder_1"
+        }
+
+    },
+
+    "overrides": {
```

Comment:
Looking at the [product page](https://hellbot.xyz/magna-2/), you might want to consider setting the `extruders_share_nozzle` and `extruders_share_heater` settings to `true`?